module/os/freebsd/zfs/sysctl_os.c

```diff
@@ -679,6 +679,27 @@ param_set_deadman_failmode(SYSCTL_HANDLER_ARGS)
 	return (-param_set_deadman_failmode_common(buf));
 }
 
+int
+param_set_raidz_impl(SYSCTL_HANDLER_ARGS)
+{
+	const size_t bufsize = 128;
+	char *buf;
+	int rc;
+
+	buf = malloc(bufsize, M_SOLARIS, M_NOWAIT | M_ZERO);
```

Comment:
Why do you use `M_NOWAIT here`?  We are not under lock here or in some fancy context.  Plus you do not check the for errors, that are possible with `M_NOWAIT`.  Lets change to `M_WAITOK`.